lib/rubocop/cop/style/safe_navigation.rb

```diff
@@ -87,6 +87,7 @@ module Style
       #   foo.baz + bar if foo
       #   foo.bar > 2 if foo
       class SafeNavigation < Base # rubocop:disable Metrics/ClassLength
+        include IgnoredNode
```

Comment:
Can you remove this redundant `include IgnoredNode`?

```suggestion
```

Reply:
Done 👍